### ISSUE TITLE
The website forum isn't sending out confirmation emails.

### ISSUE THREAD

**Issue by JamesFalero** [nim-lang/website#128]
I've tried on three different emails to signup. After logging in, I couldn't post anything because I hadn't confirmed my email according to the system. Problem is, even with three different providers, I never received a confirmation email. I even tried deleting and recreating my account, no luck.

Might explain why the participation rate is low in the forums.

**Comment by dom96** [comment]
I just tested it and it's working (gmail). Your email provider must just be rejecting the NimForum's email.

Guessing this is a "yes", but did you check the spam folder?

**Comment by JamesFalero** [comment]
Yeah. Tried with gmail too, got nothing, must be on my end, thanks.

**Comment by dom96** [comment]
You can try whitelisting "``forum[at]nim-lang.org``", might help
